### PR TITLE
Fix a problem on ZCML loading depending on how the policy package is …

### PR DESCRIPTION
**Pull request by sneridagh**
…named, related to the load of permissions in control panels and multilingual.

Difficult create a test for it...

### CODE REVIEW

**Comment by sneridagh** [comment]
Related to #526.

**Comment by sneridagh** [comment]
@mauritsvanrees @pbauer @esteele 
5.2 fails, it doesn't find `plone.app.controlpanel`, is it possible we do not load it by default in tests anymore? It's a problem if we need to support it in both 4.3 and 5.2.

https://travis-ci.org/plone/plone.restapi/jobs/505331283#L1400

I can add a zcml condition, but not ideal...

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
ok, the last commit did the trick.

File: src/plone/restapi/services/multilingual/configure.zcml [hunk 1]
```diff
@@ -3,6 +3,8 @@
     xmlns:plone="http://namespaces.plone.org/plone"
     xmlns:zcml="http://namespaces.zope.org/zcml">
 
+    <include package="plone.app.multilingual" file="permissions.zcml"/>
```
Comment (tisto) [inline src/plone/restapi/services/multilingual/configure.zcml]: @sneridagh don't we have to add p.a.multilingual to setup.py as well then? I mean at least for Plone 4? Or we wrap it in a conditional zcml...

Comment (tisto) [reply src/plone/restapi/services/multilingual/configure.zcml]: ok, it is already wrapped properly: https://github.com/plone/plone.restapi/blob/55bfaf38c62a041c18ca94b60a703c5eacecbbee/src/plone/restapi/services/configure.zcml#L29

Comment (sneridagh) [reply src/plone/restapi/services/multilingual/configure.zcml]: @tisto it is already wrapped https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/configure.zcml#L29

**Comment by pbauer** [comment]
Nonono! This pulls `plone.app.controlpanel` into Plone but it is no longer a dependency. Please make it conditional. This broke the coredev:

```
Uninstalling test.
Uninstalling instance.
Unused options for buildout: 'config-directory' 'template-directory'.
Updating _mr.developer.
Installing instance.
Getting distribution for 'plone.app.controlpanel'.
warning: no previously-included files matching '*pyc' found anywhere in distribution
Got plone.app.controlpanel 4.0.0.
While:
  Installing instance.
  Getting distribution for 'plone.app.controlpanel'.
Error: Picked: plone.app.controlpanel = 4.0.0
```

**Comment by sneridagh** [comment]
Then we have a problem because we have to make it conditional only for 5.2. And then, in ZCML we have to lookup in two different places depending on if it's 5.2 or not. Do we have some way to look that up?

**Comment by mauritsvanrees** [comment]
We can use `zcml:condition="have plone-52"`. See for example [`plone.app.upgrade`](https://github.com/plone/plone.app.upgrade/blob/2.0.22/plone/app/upgrade/configure.zcml#L11-L13).

But wouldn't `zcml:condition="installed plone.app.controlpanel"` be enough? So something like this (untested):

```
   <include package="plone.app.controlpanel"
        file="permissions.zcml"
        zcml:condition="installed plone.app.controlpanel"/>
 ```

**Comment by sneridagh** [comment]
@mauritsvanrees Fixed in #697, waiting for merge

**Comment by mauritsvanrees** [comment]
Ah, I only saw a link to issue #696 above, and did not notice the PR #697 yet.
Approved and merged. Thanks!

**Pull request merged**